isb_web/config.py

```diff
@@ -69,6 +69,17 @@ class Settings(BaseSettings):
     # The authority id used for samples directly created in the iSB instance
     authority_id: str = "iSamples"
 
+    # The absolute path to the FastText model
+    fasttext_model_path: str = "UNSET"
```

Comment:
we would want to integrate the .env file in the isamples_metadata into isb_web_config.env based on #215.